docs/OV_Runtime_UG/hetero_execution.md

```diff
@@ -1,70 +1,74 @@
-Automatic policy does not take into account device peculiarities such as the inability to infer some operations without other special operations placed before or after that layer. The plugin is responsible for solving such cases. If the device plugin does not support the subgraph topology constructed by the HETERO device, then you should set affinity manually.
-- Manual policy assumes explicit setting of affinities for all operations in the model using the runtime information ov::Node::get_rt_info.
+These two steps are not interconnected and affinities can be set in either of two ways (as described below): in the `manual` mode or using the `automatic fallback` policy.
 
 ### Defining and Configuring the Hetero Device
 
-Following the OpenVINO™ convention of labeling devices, the Hetero execution uses the name `"HETERO"`. Configuration options for the Hetero device:
+Following the OpenVINO™ naming convention, the Hetero execution plugin is assigned the label of `"HETERO".` It may be defined with no additional parameters, resulting in defaults being used, or configured further with the following setup options: 
 
-| Parameter name | C++ property | Parameter values | Default | Description |
-| -------------- | ---------------- | ---------------- | --- | --- |
-| "MULTI_DEVICE_PRIORITIES" | `ov::device::priorities` | comma-separated device names with no spaces | N/A | Prioritized list of devices |
++-------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Parameter Name & C++ property | Property values                            | Description                                               |
++===============================+============================================+===========================================================+
+| | "MULTI_DEVICE_PRIORITIES"   | | HETERO: <device names>                   | | Lists the devices available for selection.              |
+| | `ov::device::priorities`    | | comma-separated, no spaces               | | The device sequence will be taken as priority           |
+| |                             | |                                          | | from high to low.                                       |
++-------------------------------+--------------------------------------------+-----------------------------------------------------------+
 
-### Automatic and manual policies for assigning affinities
 
-`Automatic fallback` policy decides which operation goes to which device automatically according to the support in dedicated devices (`GPU`, `CPU`, `MYRIAD`, etc) and query model step is called implicitly by Hetero device during model compilation:
+### Manual and Automatic policies for assigning affinities
+
+**The manual policy** assumes setting affinities explicitly for all operations in the model using `ov::Node::get_rt_info` with the `"affinity"` key. 
```

Comment:
Manual mode is more complex, so the auto mode should come first in docs because it's applicable for most of the users.

Reply:
The manual mode description is shorter (even though the operation itself is more complex) and this solution works in every case. The automatic option's description is more complex, has some exceptions, and refers to the manual mode for cases in which it does not work. I think the reader will have a better picture reading this if they get the manual info before. And with so little text, the "manual" information might look as an afterthought when placed after the automatic one. Just a proposition though.

Reply:
let's have a call

Reply:
sure thing, whenever you have time, just let me know.